File: Pinball App Android/app/src/main/java/com/pillyliu/pinprofandroid/standings/StandingsModels.kt
package com.pillyliu.pinprofandroid.standings

internal data class StandingsCsvRow(
    val season: Int,
    val player: String,
    val total: Double,
    val rank: Int?,
    val eligible: String,
    val nights: String,
    val banks: List<Double>,
)

internal data class Standing(
    val rawPlayer: String,
    val seasonTotal: Double,
    val eligible: String,
    val nights: String,
    val banks: List<Double>,
)

internal data class StandingsWidths(
    val rank: Int,
    val player: Int,
    val points: Int,
    val eligible: Int,
    val nights: Int,
    val bank: Int,
)

File: Pinball App 2/Pinball App 2/standings/StandingsModels.swift
import Foundation

struct Standing: Identifiable {
    let id: String
    let rawPlayer: String
    let seasonTotal: Double
    let eligible: String
    let nights: String
    let banks: [Double]
}

struct StandingsCSVRow {
    let season: Int
    let player: String
    let total: Double
    let rank: Int?
    let eligible: String
    let nights: String
    let banks: [Double]
}
